PDF/Viewer/IPDFViewer.SuperMemo.cs

```diff
@@ -142,8 +144,14 @@ protected bool CreateSMExtract()
 
         contents.Add(new TextContent(true, text));
       }
-      else if (Config.ImageExtractAddHtml)
+      else if (Config.ImageExtractAddHtml && imgExtracts.Count > 0)
       {
+        extractTitle = $"image extract: {imgExtracts.Count} image{(imgExtracts.Count == 1 ? "" : "s")} from ";
+        foreach(var pi in pageIndices)
```

Comment:
Can you use `string.Join(...)` instead ? It will be much cleaner and remove the need for `TrimEnd`